bin/lib/Logos/Generator/Base/Function.pm

```diff
@@ -2,6 +2,8 @@ package Logos::Generator::Base::Function;
 use strict;
 use Logos::Generator;
 use Logos::Util;
+use parent qw(Logos::Generator::Base::Common);
+
```

Comment:
![image](https://github.com/user-attachments/assets/62f22369-4c5e-420c-832a-0f0a53d12f6a)

Reply:
I would have been thrilled to receive and address any substantive feedback from all reviewers

I’m not coming back to this after months for a whitespace (respectfully)